### DISCUSSION TITLE
Bounding box tightness

### DISCUSSION THREAD

**Discussion by timmermansjoy** [ultralytics/yolov5#10813] [Q&A]
Hi there. We here at the team are having some discussions about bounding box tightness for training. are there any guidelines around this about how tight a bounding box should be. I get the most optimal is the middle one, but on a giant dataset of videos, if one frame does not fully match up like the first image. does this have any performance problems? or is time better spend labeling more data then fixing these small annotations?

![Screenshot 2023-01-23 at 11 26 07 AM](https://user-images.githubusercontent.com/61321383/214018057-c5ada0bb-3b0b-4e82-915a-1c6afd3de894.png)

![Screenshot 2023-01-23 at 11 26 02 AM](https://user-images.githubusercontent.com/61321383/214018060-a64a95b2-64b9-4316-9f12-f6accacec7e1.png)

![Screenshot 2023-01-23 at 11 26 14 AM](https://user-images.githubusercontent.com/61321383/214018050-508a4775-7655-4ed2-9bc8-a663f228aaaa.png)

**Comment by erdesigns-eu** [comment]
I would like to know this too.. Does tighter annotations lead to better predictions?

**Reply by healthmatrice** [reply]
to my experience I doubt you can do much. Even with quite tight annotations. Basically you try to improve the regression result here. Tighter box sure helps but to a degree, you'd better check what is the percentage of  the offset error it is. In my opinion as long as you do not have systematic bias (all your box a much bigger than objects, or all your box are left offset, etc), the noise in your annotation does not matter much. Think about how the linear regressions are done when you have noisy data.

But if you are about to increase the classification performance. There are many way to improve the learning are better than lightly tighter GT I think. For example, decouple the regression/classification headers, you can adopt a similar way as PAA to increase the positive sample rate: such as assign all bbox around the GT center to be positive.
